plugins/00_dokku-standard/commands

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
-source "$PLUGIN_PATH/common/functions"
-source "$PLUGIN_PATH/config/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/config/functions"
```

Comment:
By default `$PLUGIN_AVAILABLE_PATH/config/function` and `$PLUGIN_CORE_AVAILABLE_PATH/config/function` are the same file because of symlinks. However `$PLUGIN_AVAILABLE_PATH` and `$PLUGIN_CORE_AVAILABLE_PATH` are different. So if you modify the directory contents, then yeah, you could override everything by the `common` plugin.